L1Trigger/L1TMuonEndCap/src/SectorProcessorLUT.cc

```diff
@@ -27,6 +27,8 @@ void SectorProcessorLUT::read(bool pc_lut_data, int pc_lut_version) {
     coord_lut_dir = "ph_lut_v3_data";  // Update in September 2017 from ReReco alignment, data only
   else if (pc_lut_version == 3 && pc_lut_data)
     coord_lut_dir = "ph_lut_Run3_2022_data";  // Update in October 2022 from Run 3 2022 alignment, data only
+  else if (pc_lut_version == 4 && pc_lut_data)
+    coord_lut_dir = "ph_lut_Run3_2023_data";  // Update in June 2023 from Run 3 2023 alignment, data only
```

Comment:
The PR tests were run without https://github.com/cms-data/L1Trigger-L1TMuon/pull/25, and they did not crash.
Evidently the new version is never tested.
How can we actually test it? Could you at lease please show a log of a workflow run on some data posterior to July 1st, so that we can verify that this does not crash with them?

Reply:
Yes, this is a problem with testing these PRs. I'm adding a brief log from re-emulating a recent run. I changed the LogInfo from https://github.com/cms-sw/cmssw/blob/31938879cbb9b6a64df6e29e15374d17274cce7e/L1Trigger/L1TMuonEndCap/src/SectorProcessorLUT.cc#L18 to LogWarning and you can see that in the log as well. Please let me know if you want more validation. Thanks! 

```
11-Jul-2023 08:29:46 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root
%MSG-w XrdAdaptorInternal:  file_open 11-Jul-2023 08:29:47 CEST pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root?xrdcl.requuid=60647035-ea8c-4468-9f6e-06611d8297a4.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 11-Jul-2023 08:29:47 CEST pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root?tried=&xrdcl.requuid=10be2b5d-6604-4948-8898-ee157a341e1b.
%MSG
11-Jul-2023 08:29:47 CEST  Initiating request to open file root://xrootd-cms.infn.it//store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root
11-Jul-2023 08:29:49 CEST  Successfully opened file root://xrootd-cms.infn.it//store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root
Begin processing the 1st record. Run 370332, Event 593319087, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:54.150 CEST
%MSG-w L1T:  L1TMuonEndCapTrackProducer:simEmtfDigisData  11-Jul-2023 08:29:56 CEST Run: 370332 Event: 593319087
EMTF using pc_lut_ver: 4, configured for data
%MSG
2023-07-11 08:29:56.664644: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-07-11 08:29:56.665110: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: UNKNOWN ERROR (34)
2023-07-11 08:29:56.665150: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (lxplus724.cern.ch): /proc/driver/nvidia/version does not exist
Begin processing the 2nd record. Run 370332, Event 593411623, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:56.838 CEST
Begin processing the 3rd record. Run 370332, Event 593669958, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:56.919 CEST
Begin processing the 4th record. Run 370332, Event 593989761, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:56.988 CEST
Begin processing the 5th record. Run 370332, Event 593409302, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.050 CEST
Begin processing the 6th record. Run 370332, Event 593839778, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.107 CEST
Begin processing the 7th record. Run 370332, Event 594204673, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.190 CEST
Begin processing the 8th record. Run 370332, Event 593438427, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.267 CEST
Begin processing the 9th record. Run 370332, Event 593796134, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.346 CEST
Begin processing the 10th record. Run 370332, Event 593326056, LumiSection 260 on stream 0 at 11-Jul-2023 08:29:57.410 CEST
11-Jul-2023 08:29:57 CEST  Closed file root://xrootd-cms.infn.it//store/data/Run2023D/Muon0/RAW/v1/000/370/332/00000/001535af-e2f9-407a-8e08-16c5b61a2b6b.root
```